packages/teleterm/src/ui/DocumentCluster/ClusterResources/Databases/useDatabases.ts

```diff
@@ -14,51 +14,24 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-import { AttemptStatus } from 'shared/hooks/useAsync';
-
 import { useAppContext } from 'teleterm/ui/appContextProvider';
 import { Database, ServerSideParams } from 'teleterm/services/tshd/types';
 import { routing } from 'teleterm/ui/uri';
-import { GatewayProtocol } from 'teleterm/ui/services/clusters';
+import { GatewayProtocol, makeDatabase } from 'teleterm/ui/services/clusters';
 
 import { useServerSideResources } from '../useServerSideResources';
-import { useClusterContext } from '../../clusterContext';
-
-function getEmptyTableText(status: AttemptStatus) {
-  switch (status) {
-    case 'error':
-      return 'Failed to fetch databases.';
-    case '':
-      return 'Searching…';
-    case 'processing':
-      return 'Searching…';
-    case 'success':
-      return 'No databases found.';
-  }
-}
 
 export function useDatabases() {
   const appContext = useAppContext();
-  const clusterContext = useClusterContext();
-  const documentUri = clusterContext.documentUri;
 
-  const {
-    fetchAttempt,
-    updateQuery,
-    agentFilter,
-    prevPage,
-    nextPage,
-    updateSearch,
-    onAgentLabelClick,
-    updateSort,
-    pageCount,
-  } = useServerSideResources<Database>(
-    { fieldName: 'name', dir: 'ASC' }, // default sort
-    (params: ServerSideParams) =>
-      appContext.resourcesService.fetchDatabases(params)
-  );
+  const { fetchAttempt, documentUri, ...serverSideResources } =
```

Comment:
Minor nit but `documentUri` doesn't seem to be used anywhere, right? It's used within `useServerSideResources` but it doesn't seem that other places need it at the moment so it probably could not be exposed in the first place.

Reply:
I can remove it. I think we originally used it for the old `retryWithRelogin` and it has become vestigial